src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -113,7 +113,7 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.attributes.last_triggered}
                   ></ha-relative-time>
                 `
-              : ""}
+              : "never"}
```

Comment:
Please use localization

Reply:
You now show never when there is anything else than `last-triggered` in the config for `secondary-info`

Reply:
feared as much, that's not what I wanted of course... sorry.
Am afraid this is above my current skill set... could you give me a hint please? Do we need a test on last_triggered to exist, ie !== null ?

like:

              : this.config.secondary_info === "last-triggered"
                :stateObj.attributes.last_triggered !== null
                ? html`
                    <ha-relative-time
                      .hass=${this.hass}
                      .datetime=${stateObj.attributes.last_triggered}
                     ></ha-relative-time>
                  `
                : "never"
same goes for the localization. I need to have an example so I can check. Please let me know if I can change this in the PR like this. thanks.

Reply:
```suggestion
              : this.config.secondary_info === "last-triggered"
              ? stateObj.attributes.last_triggered
              ? html`
                  <ha-relative-time
                    .hass=${this.hass}
                    .datetime=${stateObj.attributes.last_triggered}
                  ></ha-relative-time>
                `
              : this.hass.localize("ui.panel.lovelace.entities.never_last_triggered")
              : ""
```
You'll need to add the translation field to [src/translationsen.json](https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/translations/en.json) and probably come up with a better key value name.

Reply:
thanks Ian!
id like to keep it short and simple, so prefer "never". wouldn't that suffice? anyone adding last-triggered for secondary_info: would know what that means.